core/tests/protractor_utils/waitFor.js

```diff
@@ -125,7 +125,8 @@ var newTabToBeCreated = async function(errorMessage, urlToMatch) {
  */
 
 var fileToBeDownloaded = async function(filename) {
-  var name = DPATH + '/' + filename;
+  var name = Constants.protractorConstants.DOWNLOAD_PATH + '/' + filename;
+  name = path.resolve(__dirname, name);
```

Comment:
Here we end up calling `path.resolve(__dirname, ...)` twice since we also call it in ProtractorConstants.js to create `DOWNLOAD_PATH`. I don't think that's going to give you the right path, or at least it seems unnecessary